rpms.in.yaml

```diff
@@ -0,0 +1,18 @@
+packages:
+# builder stage in image/rhel/konflux.Dockerfile
+- jq
+# final stage in image/rhel/konflux.Dockerfile
+- findutils
+- postgresql
+moduleEnable:
+# final stage in image/rhel/konflux.Dockerfile
+- postgresql:15
+contentOrigin:
+  repofiles: [ "rpms.rhel.repo" ]
+context:
+  bare: true
```

Comment:
Please add a comment what `bare: true` entails.

Reply:
`bare: true` is itself documented (as `--bare`) in https://github.com/konflux-ci/rpm-lockfile-prototype/blob/main/README.md so I thought that giving attention to each attribute could be too much but instead I suggest referencing our overall doc <https://spaces.redhat.com/display/StackRox/How+to+prefetch+RPMs+for+ACS+Konflux+builds> which I'll update and offer for your review. How does it sound?

Reply:
sounds good to me.